recipes/protobuf/config.yml

```diff
@@ -1,4 +1,6 @@
 versions:
+  "4.23.1":
+    folder: all
```

Comment:
Versions 3.23.1 and 4.23.1 should be equal from source point of view. Why adding higher one?
![image](https://github.com/conan-io/conan-center-index/assets/11200321/7ec6ae5f-3377-4149-b61a-a4243c6133b7)
![image](https://github.com/conan-io/conan-center-index/assets/11200321/ca440b42-c116-4c18-a37a-d8cad411febd)

Reply:
Why do we in general use 3 or 4 prefix? I.e. grpc doesn't use it to specify dependencies
![image](https://github.com/conan-io/conan-center-index/assets/11200321/aaf2c81c-6b5f-4619-8bb7-7315c3668929)